core/cogs/Misc.py

```diff
@@ -0,0 +1,132 @@
+import discord
```

Comment:
## pt

tem como utilizar o nome do arquivo em letra minuscula?

Por padrao em python, nomes de modulos sempre são em `snake_case`

Sugestão `Misc.py` -> `misc.py`

## en

Is it possible to rename the module name to lowercase?

In python standartds we write module names n snake_case

Suggestion: `Misc.py` -> `misc.py`


From PEP8:

> "Python packages should also have short, all-lowercase names"

### Reference

https://www.python.org/dev/peps/pep-0008/#package-and-module-names